transit/services/shipment_add_costs_service.py

```diff
@@ -0,0 +1,23 @@
+from django.db import transaction
+
+from transit.models.shipment import ShipmentDetails
+
+
+class ShipmentAddCostsService:
+    """
+        Service allowing to add costs to the shipments
+    """
+
+    @transaction.atomic
+    def add_cost_to_shipment(self, shipment):
+        """
+            Save costs to given shipment.
+
+            :param shipment: shipment to which costs will be assigned.
+        """
+
+        ShipmentDetails.objects.filter(id=shipment['id']).update(
```

Comment:
ID is unique, there should be get, also it'll be better to pass shipment object as an argument. We shouldn't expect some specific payload with ['id'], please find examples from other services.